contracts/mock/base/ConsensusModifiersMock.sol

```diff
@@ -16,10 +16,10 @@
 pragma solidity 0.5.17;
```

Comment:
Please add LoanTermsConsensusMock, LoansModifiersMock and ConsensusModifiersMock in the truffle-config.js, section: mocha.reporterOptions.excludeContracts.